### PR TITLE
feat: notifies presence of newer results

### PR DESCRIPTION
**Pull request by cesarenaldi**
This is a proof of concept to gather some consensus on the approach. If happy I'll make all the necessary changes to all read hooks.

### CODE REVIEW

File: examples/web-wagmi/src/publications/UsePublications.tsx [hunk 1]
```diff
@@ -62,6 +64,12 @@ export function UsePublications() {
           ))}
         </fieldset>
 
+        {beforeCount > 0 && (
+          <button disabled={loading} onClick={prev}>
```
Comment (desfero) [inline examples/web-wagmi/src/publications/UsePublications.tsx:68]: I'm not sure without running it locally and debugging, does `loading` flag switches to `true` when you call `prev`?

Comment (cesarenaldi) [reply examples/web-wagmi/src/publications/UsePublications.tsx:68]: mmmhh.. good question... in my tests turns out it wasn't even doing the request... it was using the cache... will test

Comment (cesarenaldi) [reply examples/web-wagmi/src/publications/UsePublications.tsx:68]: checked... `loading` is stable!

File: packages/react/src/helpers/reads.ts [hunk 2]
```diff
@@ -132,18 +156,72 @@ type InferPaginatedItemsType<T extends PaginatedQueryData<unknown>> = T extends
...
+    async function (prev: Cursor) {
+      const response = await fetch({
+        variables: {
+          ...variables,
+          cursor: prev,
+        } as TVariables,
+      });
+
+      if (response.data) {
+        setBeforeCount(response.data.result.items.length);
```
Comment (krzysu) [inline packages/react/src/helpers/reads.ts:200]: I don't get how `beforeCount` is set. I tested the new example in `UsePublications` and button never appeared for me when creating a new post from another app.
The `prev` call works as expected, fetching newer posts. but `beforeCount` never changes.

**Review by krzysu — APPROVED** [review]
in general looks good, left one question

File: packages/react/src/helpers/reads.ts (hunk 2 above)
Comment (krzysu) [reply packages/react/src/helpers/reads.ts:200]: ok, after reading the original issue thread (https://github.com/lens-protocol/lens-sdk/issues/419), all clear now, it's good to go 👍

**Pull request merged**